backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHMetricsApi.scala

```diff
@@ -53,6 +53,9 @@ class CHMetricsApi extends MetricsApi with Logging with LogLevelUtil {
       "extraTime" -> SQLMetrics.createTimingMetric(sparkContext, "extra operators time")
     )
 
+  override def genBatchScanTransformerMetricsUpdater(
+      metrics: Map[String, SQLMetric]): MetricsUpdater = new BatchScanMetricsUpdater(metrics)
+
```

Comment:
why make this change ? it seems not necessary

Reply:
make `genBatchScanTransformerMetricsUpdater` and `genBatchScanTransformerMetrics` closer in proximity.